mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -163,6 +163,10 @@ class LogisticRegressionModel (
   }
 
   override protected def formatVersion: String = "1.0"
+
+  override def toString: String = {
+    super.toString + s", numClasses = $numClasses, threshold = ${threshold}"
```

Comment:
Trivial: this line uses both braces and no-braces for variables the interpolation. It's not needed in either case. The superclass `toString` is like `(weights=..., intercept=...)` so it feels a little inconsistent to continue it with `, numClasses = ...`. How about making the superclass method consistent with this?